spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/filters/ZuulProperties.java

```diff
@@ -244,8 +244,11 @@ public void init() {
 
 		private boolean customSensitiveHeaders = false;
 
+		private Set<String> methods = new LinkedHashSet<>();
```

Comment:
Hi @spencergibb, what are you views on `Route` object as a wrapper/decorator over `ZuulRoute` instead of populating its own fields?